spec/PHPeg/Generator/ToClassVisitorSpec.php

```diff
@@ -230,7 +230,13 @@ protected function parseFoo()
 
     private function line()
```

Comment:
This method name is somewhat misleading, but I'm not sure what I want to do with it.